apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -176,23 +176,48 @@ const TutorialExplorer = React.createClass({
     }
   },
 
+  /**
+   * Given a sort by choice (popularityrank or displayweight) and a grade range,
+   * return the field name from the tutorials data that should used for sorting.
+   */
+  getSortByFieldName(sortBy, grade) {
+    let sortByFieldName;
+
+    // If we're sorting by recommendation (a.k.a. displayweight) then find the
+    // right set of data to match the currently-selected grade.
+    if (sortBy === TutorialsSortBy.displayweight) {
+      if (grade === "all") {
+        sortByFieldName = "displayweight";
```

Comment:
Can we use the enum value here?  (Adding the new values to the enum as well.)

```js
sortByFieldName = TutorialsSortBy.displayweight;
...
sortByFieldName = TutorialsSortBy.displayweight_k5;
```

Reply:
Good point.  There's a slight disconnect because the enum is designed to correlate to values in the (temporarily removed) dropdown, versus the actual field names that we use.  I think I'll just introduce a second enum to address this.